wiki/Do_you_really_want_to_ask_peppy/en.md

```diff
@@ -12,4 +12,4 @@ For account issues, problems with payments, ban follow-ups, enquiries about tour
 
 ## Contacting peppy
 
-If you wish to contact peppy personally for non-osu! related reasons, you can email him at [pe@ppy.sh](mailto:pe@ppy.sh). Please be aware that any support or account-related requests to this email will be marked as spam.
+If you wish to contact peppy personally for non-osu! related reasons, you can email him at [pe@ppy.sh](mailto:pe@ppy.sh) or Chat `#general` in [osu! Discord Server](/wiki/Community/osu!_Discord_server). Please be aware that any support or account-related requests to this email will be marked as spam.
```

Comment:
```suggestion
If you wish to contact peppy personally for non-osu! related reasons, you can email him at [pe@ppy.sh](mailto:pe@ppy.sh). Please be aware that any support or account-related requests to this email will be marked as spam.
```

the osu discord is not a good way to get in contact with peppy personally, this isn't a good thing to encourage

Reply:
okay i'll delete it sry for guess it